### PR TITLE
CEDS-1499 content review

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: app/assets/stylesheets/customsdecexfrontend-app.scss [hunk 1]
```diff
@@ -32,3 +32,7 @@ a.button--gray, button.button--gray, .button--gray:visited {
 .mb-3 {
   margin-bottom: 3rem;
 }
+
+.margin-or {
```
Comment (ed0906) [inline app/assets/stylesheets/customsdecexfrontend-app.scss]: margin or?

File: app/views/components/address.scala.html [hunk 2]
```diff
@@ -43,10 +43,11 @@
         label = messages("supplementary.address.postCode")
     )
 
-    @components.fields.field_autocomplete(
+    @components.fields.field_autocomplete_with_labelclass(
```
Comment (ed0906) [inline app/views/components/address.scala.html]: Couldnt we just add an optional label class to the existing autocomplete?

File: app/assets/stylesheets/customsdecexfrontend-app.scss (hunk 1 above)
Comment (SteveSugden) [reply app/assets/stylesheets/customsdecexfrontend-app.scss]: Its used to create space around the content "Or".

File: app/views/declaration/add_transport_containers.scala.html [hunk 3]
```diff
@@ -44,8 +44,6 @@
             )
         }
 
-        <br/>
```
Comment (ed0906) [inline app/views/declaration/add_transport_containers.scala.html]: 👍

File: app/assets/stylesheets/customsdecexfrontend-app.scss (hunk 1 above)
Comment (ed0906) [reply app/assets/stylesheets/customsdecexfrontend-app.scss]: Can we name it similar to the others in here e.g. `mt-5`

File: app/views/cancel_declaration.scala.html [hunk 4]
```diff
@@ -57,7 +57,7 @@
             labelClass = Some("bold-small")
         )
 
-        @components.fields.field_radio_with_hint(
```
Comment (ed0906) [inline app/views/cancel_declaration.scala.html]: 👍

File: app/views/components/address.scala.html (hunk 2 above)
Comment (SteveSugden) [reply app/views/components/address.scala.html]: Thought of that.  Existing component has var-args as last param which precludes default arg value and most of the places that use that component (lots) don't use named arg lists.

Thought this was the simplest solution

Comment (ed0906) [reply app/views/components/address.scala.html]: 15 usages would need to be fixed, thats not _thaaaat_ many

File: app/views/components/fields/removable_elements_table.scala.html [hunk 5]
```diff
@@ -23,12 +23,14 @@
 
 <div class="field-group mb-2">
     <table id="@{id}">
-      @title.map { heading =>
-          <thead>
-              <tr>
-                <th id="@{id}-heading" colspan="2">@heading</th>
-              </tr>
-          </thead>
+      @if(elements.nonEmpty){
```
Comment (ed0906) [inline app/views/components/fields/removable_elements_table.scala.html]: So if the elements are empty we dont want to show the table title?

Would it not be better to have this `if` block around the entire `table/div`?

Comment (SteveSugden) [reply app/views/components/fields/removable_elements_table.scala.html]: Maybe around the table.  Getting rid of the div might affect the layout/spacing?   Also, there might be tests relying on finding a table by id.   

Thought this was the least invasive change

Comment (ed0906) [reply app/views/components/fields/removable_elements_table.scala.html]: I would presume the tests wouldn't look for the table by ID if it shouldn't be there though?

Comment (ed0906) [reply app/views/components/fields/removable_elements_table.scala.html]: I wouldn't worry about spacing, the margin/padding is on the wrong element if it is required even when the table isn't there

**Comment by SteveSugden** [comment]
test this please

**Pull request merged**